html/elements/frameset.json

```diff
@@ -0,0 +1,57 @@
+{
+  "html": {
+    "elements": {
+      "frameset": {
```

Comment:
Do we want to add the rows and cols attributes? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/frameset#Attributes

Reply:
Yes, fixed.